java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -872,6 +876,42 @@ public CompletableFuture<Long> zrangestore(
         return this.zrangestore(destination, source, rangeQuery, false);
     }
 
+    @Override
+    public CompletableFuture<String[]> zunion(
+            @NonNull WeightableKeys weightableKeys, @NonNull Aggregate aggregate) {
+        String[] arguments =
```

Comment:
you add `AGGREGATE` keyword to args here, but `WEIGHTS` in another file. I vote to move all args operations from there.